doc/source/reference/release_2.0.rst

```diff
@@ -5,11 +5,64 @@ Version 2.0 release notes and API changes
 This page includes more detailed release information and API changes from
 NetworkX 1.10 to NetworkX 2.0.
 
-Please send comments and questions to the networkx-discuss mailing list:
-<http://groups.google.com/group/networkx-discuss>.
+This is a [migration guide for people moving from 1.X to 2.0]()
+
+Please send comments and questions to the networkx-discuss [mailing list](http://groups.google.com/group/networkx-discuss).
 
 API changes
 -----------
+* [`<>`_]
```

Comment:
I will add the link of the merge PR after everything is reviewed and ready to merge.